documentation/guides/deployment.md

```diff
@@ -0,0 +1,201 @@
+# Deployments
+
+This document covers API and Frontend deployments. It does not include
+information regarding the ingestion server or our Airflow instance.
+
+## Real-time Deployment Information
+
+### Frontend
+
+| Environment | Deployment Log                                                                                                                | URL                                | Version Endpoint                                                 |
+| ----------- | ----------------------------------------------------------------------------------------------------------------------------- | ---------------------------------- | ---------------------------------------------------------------- |
+| Staging     | [Deployment Log](https://github.com/WordPress/openverse-frontend/actions/workflows/ghcr.yml?query=branch%3Amain+event%3Apush) | <https://staging.openverse.org>    | [`/version.json`](https://staging.openverse.org/version.json)    |
```

Comment:
Is this supposed to link to the archived openverse-frontend repository?

Reply:
Nope! I meant to update these to the new workflows once they got split up and forgot to come back to it. Thanks for catching it!

Reply:
Fixed in https://github.com/WordPress/openverse/pull/907/commits/c52f8a2f69cbef8fd907fa3d546bf22166436e42